adapters/tappx/tappxtest/params/race/banner.json

```diff
@@ -1,5 +1,5 @@
 {
     "tappxkey": "pub-12345-android-9876",
-    "endpoint": "PREBIDTEMPLATE",
-    "host": "test.tappx.com"
+    "endpoint": "ZZ123456PS",
```

Comment:
Just out of curiosity, would you mind sharing why you're changing `endpoint`, `host` and `url` throughout the JSON test files?

Reply:
Because this is the most similar version to real... The other I don't know when was writted, but this is better example.
We think that was failing because test.tappx.com was old version.
Yesterday we are paranoid. XD